src/common/options/osd.yaml.in

```diff
@@ -1045,6 +1045,19 @@ options:
   default: 21500
   flags:
   - runtime
+- name: osd_mclock_force_run_benchmark_on_init
+  type: bool
+  level: basic
```

Comment:
should probably be advanced, since most users won't need to use it

Reply:
Done.